src/components/navigation.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import logo from '../assets/logo.webp';
+import '../styles/navigation.scss';
+
+const Navigation = () => {
+  const [open, setOpen] = useState(true);
```

Comment:
[OPTIONAL] Would it make sense to have the "closed" state as default? This would allow the users to only open the navbar when they need it.

Reply:
[OPTIONAL] Also, would it make sense to add functionality to close the navbar when a navigation item is clicked? This would allow the user not to do it manually every time they navigate to the next route.